unstructured/partition/csv.py

```diff
@@ -19,8 +19,8 @@
 def partition_csv(
     filename: Optional[str] = None,
     file: Optional[Union[IO, SpooledTemporaryFile]] = None,
-    metadata_filename: Optional[str] = None,
```

Comment:
I'd avoid changing the kwarg order, just in case there's someone out there depending on argument order.

Reply:
Sure thing. Those were left over from earlier/forgot to change them back.